### PR TITLE
fix(runner): fix parallel testing db name duplication

### PR DESCRIPTION
**Pull request by stdrc**
Complement for https://github.com/risinglightdb/sqllogictest-rs/pull/194.

### CODE REVIEW

**Review by xxchan — APPROVED** [review]

File: sqllogictest/src/runner.rs [hunk 1]
```diff
@@ -1023,11 +1023,9 @@ impl<D: AsyncDB, M: MakeConnection<Conn = D>> Runner<D, M> {
             // for every slt file, we create a database against table conflict
             let file = file.unwrap();
             let db_name = file
-                .file_name()
-                .expect("not a valid filename")
                 .to_str()
-                .expect("not a UTF-8 filename");
-            let db_name = db_name.replace([' ', '.', '-'], "_");
+                .ok_or_else(|| anyhow!("not a UTF-8 filename"))?;
```
Comment (xxchan) [inline sqllogictest/src/runner.rs:1027]: Should not anyhow here. Returns `ParallelTestError`

Comment (xxchan) [reply sqllogictest/src/runner.rs:1027]: I'm comfortable with panicking here. Since non-UTF8 file name is quite rare.

Comment (stdrc) [reply sqllogictest/src/runner.rs]: OK.

**Comment by xxchan** [comment]
Oh. forget to bump version in this PR

**Pull request merged**